app/main.dev.ts

```diff
@@ -158,3 +159,11 @@ app.on('activate', () => {
   // dock icon is clicked and there are no other windows open.
   if (mainWindow === null) void createWindow();
 });
+
+// Internally `argon2-browser` uses wasm which is prohibited by the renderer's CSP
+ipcMain.on('derive-key', (event, { pass, salt }: Record<'pass' | 'salt', string>) => {
```

Comment:
this basically lets you circumvent the CSP? 🤔

Reply:
Yeah. The CSP-breaking code isn't attempting to running in the browser thread no CSP rules to break.